migrations/v2.js

```diff
@@ -0,0 +1,37 @@
+import { describe , whereContent, whereFromPlugin, whereToPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+function hasKey(object, key) {
+  if (!object) return false;
+  return Object.hasOwn(object, key);
+}
+
+function setObjectPathValue(object, path, value, force = false) {
+  if (!object) return;
+  const paths = path.split('.');
+  const key = paths.pop();
+  const target = paths.reduce((o, p) => {
+    if (!hasKey(o, p)) o[p] = {};
+    return o?.[p];
+  }, object);
+  if (!force && hasKey(target, key)) return;
+  target[key] = value;
+}
```

Comment:
Whilst these work perfectly, after discussing helper functions we've gone with using lodash as the standard with these functions already built in, we can therefore use _.has and _.set to save ourselves adding helper functions. I'll not raise this for each file. 

```suggestion
import _ from 'lodash'
```

Reply:
See https://github.com/adaptlearning/adapt-contrib-text/pull/81#discussion_r1954913291 to confirm we prefer to go this route.